new readme, improved test suite, removed unnecessary test file

SophieDeBenedetto opened the pull request:
@jmburges 

addresses #6, #4, #3, #1

jmburges commented on README.md:
```diff
@@ -1,49 +1,33 @@
-Hint 3. Proper indentation can be really helpful when you're dealing with complex if/else statements. It can be helpful to write a comment after each end to remind yourself which block or method it's closing. Using helper methods isn't required, but they can be helpful to break up your code so that it's easier to understand. 
+## Instructions
 
+* Take a look at `bin/jukebox`. This file is run by typing `ruby bin/jukebox`. Note that the file requires `lib/jukebox.rb`. 
```
I think we standarized on numbering instead of bullet points for instructions?

SophieDeBenedetto replied:
we are sticking w/ bullet points for now (i think is what we decided when we chatted on slack)

jmburges replied:
Yup!

jmburges commented on README.md:
```diff
@@ -1,49 +1,33 @@
+## Instructions
 
+* Take a look at `bin/jukebox`. This file is run by typing `ruby bin/jukebox`. Note that the file requires `lib/jukebox.rb`. 
+* Your `Jukebox` class should accept a list of songs upon initialization, and you need an `attr_accessor` for sogns. The `initialize` methods should set an `@songs` intance variable equal to the songs array that comes in as an argument.
```
sogns

jmburges replied:
intance

SophieDeBenedetto replied:
fixed.

jmburges commented on README.md:
```diff
@@ -1,49 +1,33 @@
 
+* Take a look at `bin/jukebox`. This file is run by typing `ruby bin/jukebox`. Note that the file requires `lib/jukebox.rb`. 
+* Your `Jukebox` class should accept a list of songs upon initialization, and you need an `attr_accessor` for sogns. The `initialize` methods should set an `@songs` intance variable equal to the songs array that comes in as an argument.
+* `Jukebox` instances should respond to a `.call` method. This method is being invoked in the `bin` file, which, you might remember, is responsible for running the code in a command line app. The `.call` method should aggregate the other methods you will write and be resonsible for enacting the program. We'll come back to this method in a bit. 
```
the `bin` *folder*

jmburges replied:
resonsible

jmburges replied:
not enacting, executing

SophieDeBenedetto replied:
fixed

jmburges commented on spec/01_jukebox_spec.rb:
```diff
@@ -70,6 +78,23 @@
     end
```
Is there any way we can make the `prints the available commands` test a bit clearer? Those regexes may be hard for students to understand. Maybe the `include` matcher? All we really want to see if the help message includes `help`, `list`, `play` and `exit`.

SophieDeBenedetto replied:
fixed.

jmburges commented on spec/01_jukebox_spec.rb:
```diff
@@ -70,6 +78,23 @@
+      jukebox.stub(:gets).and_return("help")
+      get_input_output = jukebox.get_input
+      do_input_output = capture_stdout { jukebox.do_command }
+      expect(do_input_output).to match(/^(?=.*help)(?=.*list)(?=.*play)(?=.*exit).+/m)
```
same comment as above. Any way we can do this without regex?

SophieDeBenedetto replied:
fixed.

SophieDeBenedetto commented:
@jmburges, addressed all comments.

AnnJohn commented:
@SophieDeBenedetto @jmburges Does this mean that it was merged (and so the issue should be closed) or that the issue needs to be closed? Just want to get some clarification on the workflow here

SophieDeBenedetto commented:
@AnnJohn this means that i addressed all the comments joe raised on it and am waiting for joe to review my edits and then approve it/tell me to ship it.

jmburges commented:
:shipit:
